src/main/drivers/resource.c

```diff
@@ -117,3 +120,12 @@ const char * const ownerNames[OWNER_TOTAL_COUNT] = {
     [OWNER_LPUART_RX] = "LPUART_RX",
     "GYRO_CLKIN",
 };
+
+const char *getOwnerName(resourceOwner_e owner)
+{
+    if (owner < 0 || owner >= OWNER_TOTAL_COUNT) {
```

Comment:
(This way bound checking is a bit more obvious/self contained) 
```suggestion
    if (owner < 0 || owner >= ARRAYLEN(ownerNames)) {
```